apps/src/code-studio/components/backpack/BackpackClientApi.js

```diff
@@ -23,8 +23,10 @@ export default class BackpackClientApi {
 
   saveFiles(filesJson, filenames, onError, onSuccess) {
     if (this.filesToUpload.length > 0) {
-      // save is currently in progress, return an error
-      onError(i18n.backpackSaveInProgress());
+      // save is currently in progress, return an error. Frontend should prevent multiple
```

Comment:
Could there be a method comment/signature detailing what the expected values are here? I was a little confused by "filesToUpload" meaning there is already a save in progress.

Reply:
I'll add some comments here. `this.filesToUpload` is a field on `BackpackClientApi` that tracks the file uploads currently in progress.